app/grandchallenge/algorithms/serializers.py

```diff
@@ -56,10 +56,6 @@ class Meta:
 
 class AlgorithmSerializer(serializers.ModelSerializer):
     average_duration = SerializerMethodField()
-    # TODO: Still to be addressed for optional inputs pitch
```

Comment:
Have all of these TODOs been checked and removed?

Reply:
Yes